torch/csrc/api/include/torch/nn/options/vision.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <torch/arg.h>
+#include <torch/csrc/WindowsTorchApiMacro.h>
+#include <torch/types.h>
+
+namespace torch {
+namespace nn {
+
+/// Options for a Grid Sample module.
+struct TORCH_API GridSampleOptions {
+  /// interpolation mode to calculate output values. Default: Bilinear
+  TORCH_ARG(std::string, mode) = "bilinear";
+  /// padding mode for outside grid values. Default: Zeros
+  TORCH_ARG(std::string, padding_mode) = "zeros";
+  /// Specifies perspective to pixel as point. Default: true
```

Comment:
I believe we can change it to match the Python version:
```suggestion
  /// Specifies perspective to pixel as point. Default: false
```